### ISSUE TITLE
org-present and normal/evilified state

### ISSUE THREAD

**Issue by jupl** [syl20bnr/spacemacs#7766]
#### Description :octocat:
After exiting `org-present`, when I press `ESC` while in normal state I enter evilified state.

#### Reproduction guide :beetle:
- Start Emacs (with `org` layer enabled)
- Open an `org` file
- Press `ESC` to assure still in normal state
- Start presenting with the `org-present` command
- Press `q` to exit presentation mode
- Press `ESC`

*Observed behaviour:* :eyes: :broken_heart:
Now in evilified state.

*Expected behaviour:* :heart: :smile:
Remain in normal state.

#### System Info :computer:
- OS: darwin
- Emacs: 25.1.3
- Spacemacs: 0.200.5
- Spacemacs branch: develop (rev. fea919c)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(clojure colors docker editorconfig emacs-lisp html javascript markdown org osx react restclient shell shell-scripts syntax-checking typescript vimscript yaml
         (auto-completion :variables auto-completion-return-key-behavior nil auto-completion-tab-key-behavior 'complete auto-completion-enable-help-tooltip t)
         (git :variables git-magit-status-fullscreen t)
         (version-control :variables version-control-diff-tool
                          (if myspacemacs--gui 'diff-hl 'git-gutter)
                          version-control-diff-side 'left))
```
- System configuration features: NOTIFY ACL LIBXML2 ZLIB TOOLKIT_SCROLL_BARS

**Comment by TheBB** [comment]
I get evilified, not emacs…

**Comment by jupl** [comment]
Ah yes. I had the wrong name. I don't use that state normally so I didn't know what the proper name. However, it does throw me way off because my keybinds change. And even if I enter `evil-normal-state` when I press `ESC` again i go back to evilified.

**Comment by TheBB** [comment]
https://github.com/syl20bnr/spacemacs/pull/7767

**Comment by smile13241324** [comment]
@jupl I have merged a fix for your issue #13088, would you mind checking whether this solved your issue and close the ticket if it does?

**Comment by zane** [comment]
@smile13241324 I was having the same issue. Looks fixed to me. Thank you so much!
